.gitignore

```diff
@@ -4,3 +4,4 @@ npm-debug.log
 
 __pycache__/
 /static
+.idea/
```

Comment:
I would recommend to ignore this is your global ignore files, it's not nice if we have to add ignores for every ide.

Reply:
Yeah, makes sense, thanks!